content/docs/concepts/flops/fl_basics.md

```diff
@@ -0,0 +1,92 @@
+  We use various components, including non-FL servers and clients.
+  Thus, we prefer to highlight FL components using *aggregators* and *learners*.
+{{< /callout >}}
+
+The aggregator starts the first FL training cycle.
+Each selected learner starts training its model copy exclusively using local data (2). 
+After the local training concludes, each learner possesses a uniquely trained local model (3). 
+
+{{< callout context="note" title="Composition of ML Models" icon="outline/info-circle" >}}
+  Most ML models (especially DNNs) consist of two parts.
+
+  The first part is (usually) a static lightweight model architecture.
+  It includes layer specification, training configuration, and hyperparameters like learning step sizes, loss, and activation functions.
+  Model architecture is typically static and lightweight in classic ML/FL.  
+
+  The second dynamic part is model weights and biases that get changed/optimized during training.
+  They allow the model to fulfill its intended use, such as prediction, inference, or generation tasks.
+  These weights and biases significantly contribute to a trained model’s overall size (space utilization).
+{{< /callout >}}
+
+FL components can transmit and share weights and biases instead of the entire trained model.
+We call model relevant data sent between the learners and aggregators (model) parameters and depict it with (P).
+
+{{<svg "flops/classic_fl/part2">}}
+
+The learners extract their model parameters and sent them to the aggregator.
+The aggregator now has access to these parameters but not the sensitive data used to train them.
+That is how FL can profit from sensitive data while maintaining its privacy. (4)
+
+The server aggregates these collected parameters into new global parameters and applies them to its model instance.
+Learners might possess varying amounts of data that lead to differently impactful updates.
+Learners typically also send the number of data samples they used for training to the aggregator to help it prioritize its received updates proportionally.
+In classic FL aggregation, the mean of the parameters is used for the global model.
+The result is a global model that was trained for one FL cycle. (5)
+
+The aggregator sends its global parameters back to the learners.
+The learners apply these parameters to their local model instance to make it identical to the aggregator’s global model.
+The FL training loop could terminate, and the learners or servers could use their global model copy for inference.
+Otherwise, another FL training cycle begins.
+There can be arbitrarily many FL cycles, similar to conventional training rounds in classic ML.
+FL training eventually terminates due to time/resource constraints or a failure to reach a satisfying performance.
+If not terminated, the accuracy and loss will worsen due to overfitting, assuming the available training data is finite and unchanging. (6)
+
+## Diving deeper into FL
```

Comment:
Instead of this section, we need a link card which allows people to jump to integrating FLOps in their workflows using Oakestra.